### PR TITLE
payee in manifest

### PR DESCRIPTION
**Pull request by jchoi330**
one of the payee objects wasn't being used but included in reduced. 
I added it to manifest event so we can test that flow too. Since we already have addPayee in updateManifest in the testVectors.

### CODE REVIEW

File: generate_testVectors_json.py [hunk 1]
```diff
@@ -806,9 +804,9 @@ def unhex(a):
     "reduced": {
         "manifest": protobuf_to_dict(current_manifest),
         "keycards": {
-            kc1.address: hex(newKc1.enroll_keycard.user_wallet.raw),
```
Comment (jchoi330) [inline generate_testVectors_json.py:809]: to make my life easier (more accurate to how we are saving KCs in stateManager.

File: testVectors.json [hunk 2]
```diff
@@ -928,9 +937,9 @@
...
     "keycards": {
-      "0xB8b8985e55aBEa8E36C777c28C08ECBe0104a37d": "0x00297113f51be97cd733d72b3fd7e3a536461d5b",
-      "0xE6bA71C62863DA55baFBCb4EA2099e77C7014601": "0xdd7877dcf849251312ce0d634ac80fed51567875",
-      "0x2E06D5C85Bd38e58E7d7a5FfDdc81552Ed79242f": "0x0000000000000000000000000000000000000000"
+      "0x00297113f51be97cd733d72b3fd7e3a536461d5b": "0xB8b8985e55aBEa8E36C777c28C08ECBe0104a37d",
+      "0xdd7877dcf849251312ce0d634ac80fed51567875": "0xE6bA71C62863DA55baFBCb4EA2099e77C7014601",
+      "0x0000000000000000000000000000000000000000": "0x2E06D5C85Bd38e58E7d7a5FfDdc81552Ed79242f"
     },
     "listings": [
       {
```
Comment (jchoi330) [inline testVectors.json:945]: I'm pretty sure base price for this first listing should be `0x00000000000000000000000000000000000000000000000000000000000001f4`.
I dont think

change_price = mevents.Listing(
```
    id=listing_simple.id,
    base_price=mtypes.Uint256(raw=int(123400).to_bytes(32, "big")),
)
events.append(change_price)
```  

in the gen test vector file is being calculated correctly?

File: generate_testVectors_json.py [hunk 3]
```diff
@@ -806,9 +804,9 @@ def unhex(a):
...
-            kc2.address: hex(newKc2.enroll_keycard.user_wallet.raw),
-            guestKeyPair.address: hex(zero_addr.raw),
+            hex(newKc1.enroll_keycard.user_wallet.raw):kc1.address,
+            hex(newKc2.enroll_keycard.user_wallet.raw):kc2.address,
+            hex(zero_addr.raw):guestKeyPair.address,
```
Comment (cryptix) [inline generate_testVectors_json.py:809]: how are you dealing with multiple guests then? this will be the case for clerks at least.

Comment (jchoi330) [reply generate_testVectors_json.py:809]: you mean multiple keycards to one wallet address?
the only purpose we are currently using the public keys are for event verification. We dont necessarily need to get all the KC associated with a WA for now.

Comment (cryptix) [reply generate_testVectors_json.py:809]: no, i mean: i you key them by the zero address, two guests will collide, potentially overriding the previous guest.

Comment (cryptix) [reply generate_testVectors_json.py:809]: if we really _only_ store these for event verification we might as well make them a big list/set

Comment (jchoi330) [reply generate_testVectors_json.py:809]: wait.. im getting confused by these names now. what i want is KC=>WA which its already doing before this change right?

**Pull request merged**